yantra/manager.py

```diff
@@ -24,8 +24,19 @@ class PluginContainer(object):
 
     def __init__(self, plugin_type):
         self._plugins = []
+        self._errors = {}
         self.plugin_type = plugin_type
 
+    @property
+    def errors(self):
```

Comment:
Any reason we don't just return the `_errors`? If you haven't tried to load anything then you'd expect no errors? 

Seems like this could lead to odd behaviour: like if you check to see if there are errors, it'll reload all the plugins as well. Maybe i'm not thinking hard enough.